event-schema/event_schema.go

```diff
@@ -710,6 +716,11 @@ func eventTypeIdentifier(eventType, eventIdentifier string) string {
 
 func (manager *EventSchemaManagerT) offloadEventSchemas() {
 	for {
+		if !areEventSchemasPopulated {
+			time.Sleep(time.Duration(10))
```

Comment:
time.Duration(10) would be 10 nano seconds.  Make it something like time.Sleep(1*time.Second)

Reply:
Done.